templates/emails/customer-notification-manual-renewal.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Customer Notification: Nudge for the customer to renew their subscription manually.
+ *
+ * @package WooCommerce_Subscriptions/Templates/Emails
+ * @version 1.0.0
+ */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * @hooked WC_Emails::email_header() Output the email header.
+ *
+ * @since 8.0.0
+ */
+do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
+
+	<p>
+		<?php
+			printf(
+				/* translators: %s: Customer first name */
+				esc_html__( 'Hi %s,', 'woocommerce-subscriptions' ),
+				esc_html( $subscription->get_billing_first_name() )
+			);
+			?>
+	</p>
+
+
+	<p>
+		<?php
+			echo wp_kses(
+				sprintf(
+					// translators: %1$s: name of the blog, %2$s: link to checkout payment url, note: no full stop due to url at the end
+					_x( 'Your subscription for XYZ on %1$s is about to expire. To keep receiving the goodies, renew it manually over here: %2$s', 'In customer renewal invoice email', 'woocommerce-subscriptions' ),
```

Comment:
Similar feedback to above. Expire in WooCommerce Subscriptions means end. We may prefer to use "expire" in this customer facing way because that makes sense to customers -- in which case this feedback can be ignored. 😅 

Perhaps something like

"You subscription for xyz on %1$s is nearly due for renewal. To keep receiving..."

The CTA is also not likely to work here either because the renewal order wouldn't have been created yet and so they would need to wait until the renewal is created in order to pay for the renewal. There's a renewal invoice email that would be sent for that purpose.

Reply:
This is a reminder that a manual payment is coming up and it seems to me that renewal flow with this URL `wcs_get_early_renewal_url` should work if it's enabled, can you please confirm?